nibiru/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nibiru",
+  "assets": [
+    {
+      "description": "The native token of Nibiru network",
+      "denom_units": [
+        {
+          "denom": "unibi",
+          "exponent": 0
+        },
+        {
+          "denom": "nibi",
+          "exponent": 6 
+        }
+      ],
+      "base": "unibi",
+      "name": "Nibiru",
+      "display": "nibi",
+      "symbol": "NIBI",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/nibiru.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/nibiru.svg"
+      },
+      "coingecko_id": "nibiru",
```

Comment:
This coingecko id doesn't exist. Please remove

Reply:
Fixed: https://www.coingecko.com/en/coins/nibi